types/joint.d.ts

```diff
@@ -2153,7 +2153,7 @@ export namespace shapes {
 
         type LinkAttributes = dia.Link.GenericAttributes<LinkSelectors>;
 
-        class Link extends dia.Link<LinkAttributes> {
+        class Link<A = dia.Link.Attributes> extends dia.Link<A> {
```

Comment:
Have you changed the default from `dia.Link.GenericAttributes<LinkSelectors>` to `dia.Link.Attributes`?

Reply:
oh, yeah, it was a mistake, I'll correct it

Reply:
should be good now, good catch